apps/src/templates/census2017/CensusMap.jsx

```diff
@@ -78,6 +80,13 @@ class CensusMapInfoWindow extends Component {
             </a>
           </div>
         </div>
+        {!missingCensusData && (
+          <div className="inaccuracy-link">
+            <a onClick={() => this.props.handleTakeSurveyClick(schoolDropdownOption, true)}>
+              I believe that the categorization for this school is inaccurate.
```

Comment:
i18n?

Reply:
Good question.  The other adjacent entries are hardcoded en so I stayed consistent with them for now.